source/content/debug-cache.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Debug Caching Issues
+description: Learn how to identify and resolve caching issues affecting your Pantheon sites.
+categories: [performance]
+tags: [cache, cdn, cookies]
+contributors: [rachelwhitton]
+reviewed: "2020-10-14"
+---
+
+## Before You Begin
+First, verify caching configuration to ensure anonymous caching is enabled. Then test to determine if CDN caching is working on your site:
+
+- Enable anonymous caching within Drupal, [details](/drupal-cache)
+- WordPress sites on Pantheon have anonymous caching enabled by default, [details](/wordpress-cache-plugin)
+- [Testing Global CDN Caching](/test-global-cdn-caching)
+
+If you see `Age: 0` after multiple requests, your site is not caching properly.
+
+### Cache Related Headers
+<dl>
+ <dt>cache-control</dt>
+  <dd>Determines caching behaviors for the given request, this configuration is set by WordPress and Drupal. </dd>
+ <dt>age</dt>
+  <dd>How long the content has been stored in cache. If 0, the response was produced by WordPress or Drupal and not served from cache. </dd>
+ <dt>set-cookie</dt>
+  <dd>Used to send cookies from the application to the user agent. The platform will not cache a response that contains the `set-cookie` header.</dd>
+</dl>
+
+## Debug Caching Issues
+Understand caching behavior for a given page by analyzing cache related HTTP headers from the command line with curl, for example:
+
+```bash{outputLines: 2-20}
+curl -I https://www.example.com
+HTTP/2 200
+cache-control: public, max-age=600 //highlight
+content-type: text/html; charset=UTF-8
+server: nginx
+//highlight-start
+set-cookie: cookielawinfo-checkbox-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
+set-cookie: cookielawinfo-checkbox-non-necessary=yes; expires=Thu, 20-Feb-2020 17:31:51 GMT; Max-Age=3600; path=/
+//highlight-end
+x-pantheon-styx-hostname: styx-fe1-a-789d66bff9-tztp6
+x-styx-req-id: 7f93c166-53fe-11ea-803e-b26d7703e33f
+date: Thu, 20 Feb 2020 16:31:51 GMT
+x-served-by: cache-mdw17379-MDW, cache-chi21146-CHI
+x-cache: MISS, MISS
+x-cache-hits: 0, 0
+x-timer: S1582216311.492451,VS0,VE204
+vary: Accept-Encoding, Cookie, Cookie
+age: 0 // highlight
```

Comment:
@alexfornuto the `cache-control` line and the `age` line are not additionally highlighted in the preview environment as intended - is that a supported thing, or nah?

Reply:
It is, just had to be adjusted to `highlight-line`.